k8sclient/rest_client.go

```diff
@@ -0,0 +1,31 @@
+package k8sclient
+
+import (
+	"github.com/giantswarm/microerror"
+	"k8s.io/client-go/rest"
+)
+
+func NewRestClient(config Config, group, version string) (rest.Interface, error) {
```

Comment:
IMO `k8sclient.NewRestClient` stutters. `k8sclient.New`, `k8sclient.NewRest`, `k8sclient.NewExt` would work for me.

Reply:
I also have to remove that `group, version string` - this is in Config.

Reply:
Done.